onnxconverter_common/float16.py

```diff
@@ -78,7 +78,8 @@ def make_value_info_from_tensor(tensor):
     return helper.make_tensor_value_info(tensor.name, tensor.data_type, shape)
 
 
-def convert_float_to_float16(model, min_positive_val=1e-7, max_finite_val=1e4, keep_io_types=False):
+def convert_float_to_float16(model, min_positive_val=1e-7, max_finite_val=1e4,
+                             keep_io_types=False, disable_shape_infer=False):
     '''
```

Comment:
Add a line to the documentation "Type/shape information is needed for conversion to work.  Set disable_shape_infer to true only if the model already has type/shape information for all tensors."